twake/frontend/src/app/components/OnBoarding/CompanyBillingBanner.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Typography } from 'antd';
+import Banner from '../Banner/Banner';
+import { CompanyType } from 'app/models/Company';
+import Groups from 'services/workspaces/groups.js';
+
+type PropsType = {
+  url: string;
+  companyId: string;
+};
+
+export default ({ url, companyId }: PropsType): JSX.Element => {
+  const { Link } = Typography;
+  const userGroups: { [key: string]: CompanyType } = Groups.user_groups;
+  const { role, plan } = userGroups[companyId];
+  const isCurrentUserAdminOrOwner = role === 'admin' || role === 'owner';
+  const billingStatusError = plan?.billing?.status === 'error';
+
+  const onClickLink = () => window.open(url, 'blank');
+
+  const shouldDisplayBanner = isCurrentUserAdminOrOwner && billingStatusError;
```

Comment:
This code is great  but you must change it a bit according to issue from Benoit (members should also see the banner but with another text)

Reply:
Done !